osler/core/forms.py

```diff
@@ -63,7 +71,21 @@ def __init__(self, *args, **kwargs):
         self.helper['languages'].wrap(InlineCheckboxes)
         self.helper['ethnicities'].wrap(InlineCheckboxes)
         self.helper.add_input(Submit('submit', 'Submit'))
-        self.fields['address'].widget.attrs = {'placeholder': settings.OSLER_DEFAULT_ADDRESS}
+        self.fields['address'].widget.attrs = {'placeholder': settings.OSLER_DEFAULT_ADDRESS,
+                                                'id': 'address'}
+        self.fields['city'].widget.attrs = {'id': 'city'}
```

Comment:
are these ids not prespecified by crispy forms? It seems strange we have to manually add this in. If we really do have to, you could do:

```python
for fname in ['city', 'state', 'zip_code', ...]:
    self.fields[fname].widget.attrs = {'id': fname}
```

it would be more DRY.

Reply:
Yeah, I remember being surprised that crispy form doesnt auto generate an id too, but yeah this is a cleaner way to do it. Here's the documentation on crispy forms below:

![image](https://user-images.githubusercontent.com/10227136/105079644-533a9400-5a55-11eb-85a8-c439cd3fc8d8.png)